crates/miden-testing/src/kernel_tests/tx/test_account_interface.rs

```diff
@@ -69,24 +73,31 @@ fn check_note_consumability_well_known_notes_success() -> anyhow::Result<()> {
         tx_args,
         source_manager,
     )?;
-    assert_matches!(execution_check_result, NoteAccountExecution::Success);
 
+    assert_matches!(execution_check_result, NoteConsumptionInfo { successful, failed, .. }=> {
+    assert_eq!(successful.len(), notes.len());
+    successful.iter().zip(notes.iter()).for_each(|(success, note)| {
+        assert_eq!(success, note);
+    });
+    assert!(failed.is_empty());
+    });
     Ok(())
```

Comment:
Nit: rustfmt doesn't format assert_matches! (or macros in general?), so just to note that this needs manual formatting.